tasks/webpack-dev-server.js

```diff
@@ -45,7 +43,10 @@ yarn add webpack-dev-server`);
       const devClient = [
         `webpack-dev-server/client?${protocol}://${opts.host}:${opts.port}`
       ];
-      if (opts.hot) devClient.push('webpack/hot/dev-server');
+      if (opts.hot) {
+        devClient.push('webpack/hot/dev-server');
+        webpackOptions.plugins.plush(new webpack.HotModuleReplacementPlugin());
```

Comment:
Yes. This was fixed in #127 already. And plush is a thing:
![standard-1024x1024 2](https://cloud.githubusercontent.com/assets/474603/24301358/54f7d4e8-10a7-11e7-8b94-04baa466054b.jpg)